### PR TITLE
Create a script that sets up paasta-internal custom resource definitions

### PR DESCRIPTION
**Pull request by EvanKrall**
(not for operators run by other teams). COMPINFRA-1081

### CODE REVIEW

File: paasta_tools/setup_kubernetes_internal_crd.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+                "kind": "DeployGroup",
+                "shortNames": ["dg"],
+            },
+            "validation": {
+                "openAPIV3Schema": {
+                    "type": "object",
+                    "properties": {
+                        "service": {"type": "string"},
+                        "deploy_group": {"type": "string"},
+                        "git_sha": {"type": "string"},
```
[the remark this answers is not in the document]
Comment (EvanKrall) [reply paasta_tools/setup_kubernetes_internal_crd.py:64]: oh yeah I suppose I should probably include that 😛

File: paasta_tools/setup_kubernetes_internal_crd.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        kind="CustomResourceDefinition",
+        metadata={
+            "name": "startstopcontrols.paasta.yelp.com",
+            "labels": {
+                paasta_prefixed("internal"): "true",
+            },
+        },
+        spec={
+            "group": "paasta.yelp.com",
+            "versions": [{"name": "v1", "served": True, "storage": True}],
```
Comment (jfongatyelp) [inline paasta_tools/setup_kubernetes_internal_crd.py:81]: I'm not familiar enough w/ k8s custom resources, does version matter to anything else about how we access the api?  I notice all operator workloads specify their crd's with `v1alpha1`.   Is there any significance to that vs `v1`?

**Review by jfongatyelp — APPROVED** [review]
it feels like we could add a little more validation later (e.g. we only allow start|stop for desired_state right?) but lgtm, once we add image_version

File: paasta_tools/setup_kubernetes_internal_crd.py (hunk 2 above)
Comment (EvanKrall) [reply paasta_tools/setup_kubernetes_internal_crd.py:79]: I think it doesn't matter - this is for versioning of the CRDs themselves. We can start wherever we want and increment these as we see fit. Maybe v1beta1 is a better starting point though.

**Review by jfongatyelp — APPROVED** [review]

**Pull request merged**